### DISCUSSION TITLE
What is the best server size for Personal with 50 CCU to 150 CCU attending my event?

### DISCUSSION THREAD

**Discussion by tttigo** [mozilla/hubs#3105] [Hubs Cloud Questions] [answered]
we are planning on creating an event using hubs cloud running on aws, and we're having some difficulties to decide on what will be the best server option - based on the [https://hubs.mozilla.com/docs/hubs-cloud-aws-estimated-cost-charts.html](url) and [https://hubs.mozilla.com/docs/hubs-cloud-aws-estimated-ccu-limits.html](url)

since we are not able to consider the estimate cost, we created 2 possibles scenarios, and we like to know which serverr type would be appropriate for each 

1 - 2 equal rooms with maximum of 25 active users each (50 CCU)

2 - 6 equal rooms with maximum of 25 active users each (150 CCU)

our event will happen during a month, with only 1 scene (the same scene in multiple rooms), with some specifics events during this. It has to be online for 30 days, but in some dates we will have specific actions, like a presentation, that we expect a bigger flow of users.

you guys wrote that the recommended for an event is the c4.large, but on the CCU table, it seems to be very similar to the t3.medium, which is much cheaper - we are very confused with this docs and the numbers in that

our budget only allow to use the Hubs Personal, is it ok for an event like we described?

thank you very much,
thiago

**Comment by robinkwilson** [comment, marked as answer]
Brilliant deduction. ~~IMO a Personal instance will work fine for your use case. A t3.medium sounds great for the off time, though during an event/presentation, I'd "update the stack" to a c4.large to ensure consistent performance.~~ - _Read correction below_

I'd suggest reading this part of the Minimizing Cost Documentation to understand the tools you have updating the stack in an event context: [Hubs Docs: Minimizing Costs an Event User Story](https://hubs.mozilla.com/docs/hubs-cloud-aws-costs.html#minimize-your-costs---a-user-story)

**Reply by tttigo** [reply]
Thanks Robin.

Just for understanding, how many exactly users t3.medium and c4.large takes? You mean t3.medium works for the scenario 1 (2 rooms with 50 active users), and the c4.large for scenario 2 (6 rooms with 150 active users)?

[On this table](https://hubs.mozilla.com/docs/hubs-cloud-aws-estimated-ccu-limits.html#estimated-ccu-table-1-app--1-stream-server) you say that t3.medium supports 45 Max Active CCU, single-room and 5 Max Rooms w/25 active CCU
1- 45 users on the same room?
2- what about on separate rooms?
3- can we create only 5 rooms?

and that c4.large supports 40 Max Active CCU, single-room, with 3 Max Rooms w/25 active CCU. This is less than t3.medium 

I'm sorry if i'm being dumb, but this docs are not helping me to make things clear.

**Reply by robinkwilson** [reply]
All good. That linked chart is not what you want for estimating CCU for a Personal 1 server instance. That chart uses an Enterprise instance with 1 app x 1 streaming server.

**The chart you need is:** [Hubs Docs: Estimating Personal CCU to Costs with 1 server](https://hubs.mozilla.com/docs/hubs-cloud-aws-estimated-cost-charts.html#estimating-personal--enterprise-costs-with-1-server)

Rereading this on a Personal instance 1 server:
For 50 CCU, you need around a `t3.large`. 
For 150 CCU, you should scale up to a `c5.xlarge`.

Remember: you can scale the server type up or down **LIVE** during an event if you see performance issues.
Also, these charts are **guidelines**, depending on how heavy your scene is, these could be way off.

**Reply by AlbertWijaya1** [reply]
@tttigo  Thanks for the thread! Would you mind updating on the type of server you end up using? and how did your event go? did it go well as intended? Thanks in advance!
